### PR TITLE
test PR, do not upvote

### PR DESCRIPTION
**Pull request by mullr**

### CODE REVIEW

File: src/puppetlabs/puppetdb/client.clj [hunk 1]
```diff
@@ -40,6 +40,9 @@
                             :character-encoding "UTF-8"
                             :accept :json}))))
 
+
+;; And a change here
+
```
Comment (rbrw) [inline src/puppetlabs/puppetdb/client.clj]: This is a test.

File: src/puppetlabs/puppetdb/mq.clj [hunk 2]
```diff
@@ -21,6 +21,8 @@
                                                      (metrics/keyword->metric-name
                                                        [:global] :message-persistence-time))}))
 
+;; And a change here
+
```
Comment (rbrw) [inline src/puppetlabs/puppetdb/mq.clj]: This is only a test.

File: src/puppetlabs/puppetdb/zip.clj [hunk 3]
```diff
@@ -10,6 +10,8 @@
 (defn transfer-meta [old-node new-node]
   (with-meta new-node (meta old-node)))
 
+;; In another commit
```
Comment (rbrw) [inline src/puppetlabs/puppetdb/zip.clj]: In the event of an actual emergency...

**Comment by mullr** [comment]
Rob Browning <notifications@github.com> writes:

> rbrw requested changes on this pull request.
>
>
>
>> @@ -40,6 +40,9 @@
>                              :character-encoding "UTF-8"
>                              :accept :json}))))
>  
> +
> +;; And a change here
> +
>
> This is a test.
>

This is a reply. 

>> @@ -21,6 +21,8 @@
>                                                       (metrics/keyword->metric-name
>                                                         [:global] :message-persistence-time))}))
>  
> +;; And a change here
> +
>
> This is only a test.
>

This is only a reply. 

>> @@ -10,6 +10,8 @@
>  (defn transfer-meta [old-node new-node]
>    (with-meta new-node (meta old-node)))
>  
> +;; In another commit
>
> In the event of an actual emergency...
>

In the event of an actual reply...

File: src/puppetlabs/puppetdb/client.clj [hunk 4]
```diff
@@ -40,6 +40,9 @@
...
 (defn-validated submit-catalog
   "Send the given wire-format `catalog` (associated with `host`) to a
```
Comment (rbrw) [inline src/puppetlabs/puppetdb/client.clj]: This is a comment directly on a commit during a batch review.

File: src/puppetlabs/puppetdb/zip.clj [hunk 5]
```diff
@@ -10,6 +10,8 @@
...
+
 (extend-protocol NodeTraversal
```
Comment (rbrw) [inline src/puppetlabs/puppetdb/zip.clj]: This is a second comment directly on a commit during a batch review.

File: src/puppetlabs/puppetdb/zip.clj [hunk 6]
```diff
@@ -10,6 +10,8 @@
...
```
Comment (rbrw) [inline src/puppetlabs/puppetdb/zip.clj]: This is a comment "directly" on a commit (i.e. on the new narrowed view).

**Pull request closed without merging**